aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -94,30 +92,33 @@ fn native_create_aggregator(
     ty_args: Vec<Type>,
     mut args: VecDeque<Value>,
 ) -> SafeNativeResult<SmallVec<[Value; 1]>> {
+    if !context.aggregator_api_enabled() {
```

Comment:
But all existing aggregators can still be used?

Reply:
yes, this is gradual rollout flag. We don't want to brick contracts after this flag has once been enabled.

aggregator_execution_enabled is fallback in case of issues with aggregator logic.